package.json

```diff
@@ -42,9 +42,9 @@
     "test": "npx hardhat test --network localhost",
     "test:fast": "TS_NODE_TRANSPILE_ONLY=1 npx hardhat test --network localhost --no-compile",
     "test:fast:compile": "TS_NODE_TRANSPILE_ONLY=1 npx hardhat test --network localhost",
-    "test:integration:polygon": "INTEGRATIONTEST=true VERBOSE=true yarn run test test/integration/polygon/**",
-    "test:integration:optimism": "INTEGRATIONTEST=true VERBOSE=true yarn run test test/integration/optimism/**",
-    "test:integration:ethereum": "INTEGRATIONTEST=true VERBOSE=true yarn run test test/integration/ethereum/**",
+    "test:integration:polygon": "ls test/integration/polygon; INTEGRATIONTEST=true VERBOSE=true yarn run test test/integration/polygon/**/*.spec.ts",
```

Comment:
Do we want to keep the ls commands here ? if so I suggest adding it to the ethereum command as well for consistency.

Reply:
Seems like this was removed while I was reviewing.